### ISSUE TITLE
Liked to entry checkbox

### ISSUE THREAD

**Issue by aVandalInShorts** [DeuxHuitHuit/entry_relationship_field#36]
It would be nice if a checkbox to select multiple items when you click on "Link to entry".

**Comment by nitriques** [comment]
What about selecting multiple entries with the Ctlr (cmd) key ?

**Comment by aVandalInShorts** [comment]
Yes! What if, when you click on the first field, only it is the One. Or else, if the user click in an other part of the field, it becomes selectable with the ctrl key.

**Comment by nitriques** [comment]
Hum seems counter intuitive... Ctrl + click for multiple selection is already an established pattern in Symphony.

**Comment by aVandalInShorts** [comment]
But it's the exact same behavior as when you look at a list of entries. The only difference is, is you access the entry instead of selecting it.

**Comment by nitriques** [comment]
Hum I'll think about it :)

**Comment by pixelninja** [comment]
Hello :) Bumping this one! It's a great idea, did it ever get implemented or thought about? Love this extension btw!!
